packages/jest-runtime/src/index.js

```diff
@@ -739,3 +739,5 @@ class Runtime {
 }
 
 module.exports = Runtime;
+// $FlowFixMe
```

Comment:
@cpojer should we start using `import/export` syntax?
lerna + multiple exports from a package make things pretty complicated with `require` syntax